README.md

```diff
@@ -1,133 +1,111 @@
+
+Previously adopted web APIs, in particular the [contain](https://developer.mozilla.org/en-US/docs/Web/CSS/contain) and [will-change](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change) CSS properties, add ways to specify forms of rendering [isolation](https://github.com/chrishtr/rendering/blob/master/isolation.md) or isolation hints, with the intention of them being a mechanism for the web developer to help the browser optimize rendering for the page.
+
+While these forms of isolation help, they do not guarantee that isolated content does not need to be rendered at all. Ideally there would be a way for the developer to specify that specific parts of the DOM need not be rendered, and pair that with a guarantee that when later rendered, it would not invalidate more than a small amount of style, layout or paint in the rest of the document.
+
+## Summary
+
+Three new features are proposed:
+
+* A new `rendersubtree` attribute (early draft spec [here](https://chrishtr.github.io/html/output/#the-rendersubtree-attribute)). This controls whether DOM subtrees do or do not render, and is the mechanism by which rendering work can be avoided. User-agent features may modify this attribute, causing on-demand rendering, if desired. The developer may listen to this on-demand rendering via a MutationObserver and respond to it before rendering occurs. `rendersubtree`, when present, forces `style` and `layout` containment, plus `size` containment if invisible. This ensures minimal invalidation of the rest of the document when rendering occurs.
+
+* A `content-size` attribute (early draft spec [here](http://tabatkins.github.io/specs/css-content-size/) that specifies how much space a subtree should take up. This is intended to allocate a placeholder size for content marked as invisible by `rendersubtree` (since it already has `size` containment, as mentioned above).
+
+* An `updateRendering` method on Element objects. This can be used to pre-render content within a subtree marked with `rendersubtree` as invisible to make it ready for display or measurement.
+
+## Example usage
+
+```html
+<div id=target rendersubtree="invisible" style="content-size: 200px 200px">...content...</div>
+<script>
+target.setAttribute('rendersubtree', ''); // makes #target render
+</script>
+```
+This div's subtree is not rendered (but the div itself is; this allows the div to show fallback or "loading..." affordances), and there is no need for the browser to do any rendering lifecycle phases for the subtree of the div. Custom element upgrades are not performed and resources are not loaded. The div lays out as if it had a single 200px by 200px child, which serves as a placeholder in order to take up the approximate layout size of the div's subtree. This allows page layout to be approximately correct, and preserves layout overflow size for scrolling. The browser may *not* render the content, even via a user-agent feature.
```

Comment:
> Custom element upgrades are not performed and resources are not loaded.

I think we need to update this to mention other values of rendersubtree, such as holdupgrades, and holdresourceload (not sure those are the names we agreed on though). If this is a straight up rename, then feel free to punt

Reply:
Moved them to a new example now